docs/excel/custom-functions-best-practices.md

```diff
@@ -43,28 +50,35 @@ Keep in mind the following best practices when creating custom functions in your
 
 * Do not change the value of an `id` property in the JSON metadata file after it's been associated with a corresponding JavaScript function name. You can change the function name that end users see in Excel by updating the `name` property within the JSON metadata file, but you should never change the value of an `id` property after it's been established.
 
-* In the JavaScript file, specify all custom function associations in the same location. For example, the following code sample defines two custom functions and then specifies the association information for both functions.
+* In the JavaScript file, specify a custom function association after each function. For example, the following code sample defines two custom functions and each is followed by their accompanying association information.
```

Comment:
Consider adding a second sentence like "This is done by calling the `CustomFunctions.associate` method." Maybe talk about the `associate` parameters too.